### PR TITLE
Changes to align with latest WHAT WG Streams API specification:

### PR DESCRIPTION
**Pull request by ClaesNilsson**
- Changed the name of the ReadableStream attribute “input” to “readable” and the name of the WritableStream attribute “output” to “writeable”.
- Changed link to [STREAMS] to latest WHAT WG Streams specification.
- Corrected editorial in 6.2: “Closes the TCP socket. Returns the closedPromise that was created in the UDPSocket constructor.”
- Added comment that so far we assume default strategy, i.e. the ReadableStream’s and WriteableStream’s constructor’s last argument is omitted.
- Description of the semantics for the input argument functions to the WritableStream updated to be aligned with latest version of the WHAT WG Streams API specification.
- start() returns a promise that is resolved when the TCP connection has been established and rejected if the TCP connection attempt failed.
- write(chunk) returns a promise that is resolved if writing succeeded and rejected if writing failed.
- close() unchanged
- abort() added

Similar for UDP.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1232,7 +1273,29 @@
...
+                   implementation will throw away any pending queued up chunks.    
+                   
+                   <p class="issue">
+                     If the constructor's <code>strategy</code> argument is
+                     omitted the 
+                     <a href= "https://streams.spec.whatwg.org/#default-ws-strategy">Default strategy for Writable Streams</a> 
+                     applies. Currently this means that the WriteableStream 
+                     object goes to "waiting" state after 1 chunk has been 
+                     written to the internal WriteableStream object's output
+                     buffer. This means that the application must call .wait()
```
Comment (domenic) [inline index.html]: Probably better to say "the application _should_ call .wait()". The application can continue writing, but if they do so they're disrespecting the backpressure signal.

**Comment by domenic** [comment]
Wow, this is great!! It's exactly what I was planning on doing, plus more that would probably have slipped my mind. <3

**Comment by ClaesNilsson** [comment]
Thanks for reviewing Domenic. Do you have more comments before I merge?

**Comment by domenic** [comment]
Nope, go for it!

On Oct 10, 2014, at 01:03, Claes Nilsson <notifications@github.com<mailto:notifications@github.com>> wrote:

Thanks for reviewing Domenic. Do you have more comments before I merge?

##

**Pull request merged**